readme_renderer/markdown.py

```diff
@@ -33,13 +33,13 @@
 try:
     import cmarkgfm
     from cmarkgfm.cmark import Options as cmarkgfmOptions
-    variants = {
-        "GFM": lambda raw: cmarkgfm.github_flavored_markdown_to_html(
+    variants: Dict[str, Callable[[str], str]] = {
+        "GFM": lambda raw: cast(str, cmarkgfm.github_flavored_markdown_to_html(
```

Comment:
Nifty trick, I'll have to remember it for the future. I've confirmed this was included in CPython since 3.5, so we're definitely safe there.